portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,353 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="home">
+      <a id='home'><h1>Teanna Barrett’s Portfolio</h1></a>
+      <p>Hello I’m Teanna Barrett and this is my portfolio. 
+          I'm creating this portfolio for my second week project as a part of the STEP internship.
+           I not only wanted to share my professional side, I want to personally  introduce myself. 
+           I have structured this app as a choose your own adventure of sorts. Each panel of the webpage 
+           will have some information about me and choices displayed at the bottom. As you continue to 
+           pick choices you will be able to learn specific things about me and the work I hope to do as 
+           a Computer Scientist. [Or you can choose the random button and the webpage will pick for you.] 
```

Comment:
Is there a random button?

Reply:
I decided against it for time sake. I moved the more content-heavy commits to a different branch so I will clarify it there.

Reply:
That's fine.

Typically in code reviews its a good idea to address comments in the same branch so it's easy to keep track of what still needs to be cleaned up. Another strategy is to insert a <!-- TODO --> comment in the code so that future readers know something is going to change. Within the Google codebase there are specific formatting rules for TODOs in the code - but for the OS project you can adopt whatever works for you.